default/stringtables/en.txt

```diff
@@ -16719,6 +16719,9 @@ GREAT_HAPPINESS_DESC
 ULTIMATE_HAPPINESS_DESC
 +++	Ultimate [[metertype METER_HAPPINESS]]: +[[value ULTIMATE_HAPPINESS_VAL]]
 
+INDEPENDENT_HAPPINESS_DESC
+⋄	Independent [[metertype METER_HAPPINESS]]: Stability unaffected by supply
```

Comment:
```suggestion
⋄	Independent [[metertype METER_HAPPINESS]]: Stability unaffected by supply connection
```

Reply:
![laenfa](https://user-images.githubusercontent.com/43386669/153621192-aae0d5e7-27d3-4a18-8094-e5f30e071ddf.png)

Seems to be a space limit here?

Reply:
This is probably misleading... Does it prevent all affects on stability that have a supply-connection dependence, or just some baseline ones scripted into the species itself?

Reply:
Still probably misleading, as it still suggests that a supply connection won't ever matter for this species' planets. I don't have a short simple alternative in mind that makes the distinction, though...